pkg/controller/job/job_controller_test.go

```diff
@@ -1048,6 +1049,25 @@ func TestControllerSyncJob(t *testing.T) {
 			podIndexLabelDisabled:  true,
 			expectedReady:          ptr.To[int32](0),
 		},
+		"FailureTarget=False condition added manually": {
+			parallelism: 1,
+			completions: 1,
+			activePods:  1,
+			readyPods:   1,
+			initialStatus: &jobInitialStatus{
+				active: 1,
+				startTime: func() *time.Time {
+					now := time.Now()
+					return &now
+				}(),
+				conditions: []batch.JobCondition{
+					*newCondition(batch.JobFailureTarget, v1.ConditionFalse, "ConditionAddedManually", "Testing", realClock.Now()),
```

Comment:
I would find it easier to read without the utility function actually. We don't need to supply time since this does not matter for repro.